Colours_And_Fonts_style_v1.css

```diff
@@ -226,7 +332,7 @@ img {
   z-index: 99; /* Make sure it does not overlap */
   border: none; /* Remove borders */
   outline: none; /* Remove outline */
-  background-color: #80D9FF; /* add a background color */
+  background-color: #657CCA; /* add a background color */

```

Comment:
I changed the background color from a light blue to a light purple as it matches the other colours better